cluster-api/gcp-deployer/deploy/deploy.go

```diff
@@ -52,7 +54,7 @@ func NewDeployer(provider string, configPath string) *deployer {
 			glog.Exit(fmt.Sprintf("Failed to set Kubeconfig path err %v\n", err))
 		}
 	}
-	md, err := newMachineDeployer(provider, token)
+	md, err := google.NewMachineActuator(token, nil)
```

Comment:
this variable shouldn't be `md` any longer since you aren't assigning it a deployer.

Reply:
Ha. It never was a machine deployer. Just a weird name for a factory function.

Reply:
Done.